config/config.yml

```diff
@@ -33,7 +33,8 @@ twigDebug: true
 twigAutoescape: false
 twigDefaultDateFormat: ""
 twigDefaultIntervalFormat: ""
-twigExtensions: '\Ajgl\Twig\Extension\BreakpointExtension'
```

Comment:
@slowbot - I changed this line earlier today, because it causes my dev env to crash. Marcin (@software-project ) helped me out with that, and from what I understood, he has the same issue. I'm not sure how to go about it.

Reply:
@claudioccm @software-project see this: https://github.com/Exygy/courtyard/pull/28/files/eb9ac438c0a7cdb0d3b07341aed8c8ea173eeda8#diff-b9e5131e854706bc09310a9ebb13f52f 

Prettier was causing the YML fails to save with improper formatting in a commit hook via Husky, which causes YML parsing errors. The fix was to exclude that config directory, which should be working...

As also noted in that PR, you will also need to run `compose install` locally, which will install the extension if you haven’t done that. HTH.  

**Note: I would highly recommend disabling Prettier on YML files entirely because it will likely cause issues in the future.**

Reply:
Awesome, thanks @slowbot and @jacine. Fixed here.

@slowbot, Jumbo is added as a pseudo-pattern. We might need to add a plugin to use with `@includes`. [Check this](https://patternlab.io/docs/pattern-pseudo-patterns.html#using-pseudo-patterns-as-pattern-includes) from the PatternLab Docs.